Doc/library/venv.rst

```diff
@@ -143,7 +143,8 @@ creation according to their needs, the :class:`EnvBuilder` class.
 
 .. class:: EnvBuilder(system_site_packages=False, clear=False, \
                       symlinks=False, upgrade=False, with_pip=False, \
-                      prompt=None, upgrade_deps=False)
+                      prompt=None, upgrade_deps=False,
```

Comment:
I think you need a backslash here for line continuation (I'll look into why Sphinx needs this as it's pretty ugly)
```suggestion
                      prompt=None, upgrade_deps=False, \
```

Reply:
Thanks! That seemed to resolve it.